### PR TITLE
update wording and formatting of messages text

### PR DESCRIPTION
**Pull request by EricFromCanada**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [n/a] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

-----
Fixed up any issues in user-facing text that I could find. (Settled on backticks for around inline shell commands.)

### CODE REVIEW

File: Library/Homebrew/cask/artifact/installer.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@ module ManualInstaller
         def install_phase(**)
           puts <<~EOS
             To complete the installation of Cask #{cask}, you must also
-            run the installer at
+            run the installer at:
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cask/artifact/installer.rb]: Thoughts on trimming the extra empty line here?

File: Library/Homebrew/cask/dsl/caveats.rb [hunk 2]
```diff
@@ -47,11 +47,11 @@ def eval_caveats(&block)
         next if MacOS.version < :high_sierra
 
         <<~EOS
-          To install and/or use #{@cask} you may need to enable their kernel extension in
+          To install and/or use #{@cask} you may need to enable their kernel extension in:
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cask/dsl/caveats.rb]: Similarly on empty lines here too

File: Library/Homebrew/cask/artifact/installer.rb (hunk 1 above)
Comment (EricFromCanada) [reply Library/Homebrew/cask/artifact/installer.rb]: Solid idea; seems wise to save line breaks for trailing important messages.

**Comment by MikeMcQuaid** [comment]
Brilliant work, thanks again @EricFromCanada!

**Pull request merged**